internal/cluster/distribution/eks/service.go

```diff
@@ -218,7 +222,7 @@ func NewNodePoolStatusFromCFStackStatus(cfStackStatus string) (nodePoolStatus No
 	switch {
 	case strings.HasSuffix(cfStackStatus, "_COMPLETE"):
 		if cfStackStatus == cloudformation.StackStatusDeleteComplete { // Note: CF stack is deleted, but DB entry is still existing.
-			return NodePoolStatusDeleting
+			return NodePoolStatusDeleted
```

Comment:
Shouldn't this be on its own case?

Reply:
Yeah, it could be.
Gonna update it.

Reply:
Also the delete in progress is handled as the "in progress" branch's special case.

Reply:
[Fixed](https://github.com/banzaicloud/pipeline/compare/ad95732ec9bff8610d98fb53d12908889923eecd..5c1d744e7fc86f9e1613cc1df3342d345d33ca4a), flattened the switch.